### PR TITLE
Fixes Issue No #17

### PR DESCRIPTION
**Pull request by Roshan13046**
Issue no #17 fixed.
Please accept and merge my PR.
Thank you.

### CODE REVIEW

**Review by nikochiko — COMMENTED** [review]
Hey @Roshan13046 Nice work! The code is looking good. Since Radix Sort isn't that common, can you add some comments explaining some important parts?

**Comment by nikochiko** [comment]
Just one-liners where it would seem tricky - like this variable does this, these lines do this, etc

File: Sorting/radix_sort/RadixSort.cpp.cpp [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    //array now contains sorted numbers according to current digit 
+    for (i = 0; i < n; i++) 
+        arr[i] = output[i]; 
+} 
+  
+//Raix sort function
+void radix_sort(int arr[], int n) 
+{ 
+    // Finding maximum number to know number of digits 
+    int m = getMax(arr, n);
```
Comment (Roshan13046) [inline Sorting/radix_sort/RadixSort.cpp.cpp:57]: m is for storing the max value from array

**Comment by Roshan13046** [comment]
> Just one-liners where it would seem tricky - like this variable does this, these lines do this, etc

All the necessary comments are mentioned.
Please check.
Thank you.

File: Sorting/radix_sort/RadixSort.cpp.cpp [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+void countSort(int arr[], int n, int exp) 
+{ 
+    int output[n]; // output array 
+    int i, count[10] = {0}; 
+  
+    // Storing the count of occurrences in count[]  array
+    for (i = 0; i < n; i++) 
+        count[ (arr[i]/exp)%10 ]++; 
+  
+    // Change count[i] so that count[i]
```
Comment (nikochiko) [inline Sorting/radix_sort/RadixSort.cpp.cpp:37]: @Roshan13046 I saw the comments, but they are still not clear. Can you be a little more descriptive?

File: Sorting/radix_sort/RadixSort.cpp.cpp [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+	//input array to sort
+    int arr[] = {2,3,44,88,986,8654,344,911};  
+    //size of array
+    int n = sizeof(arr)/sizeof(arr[0]); 
+    
+    //calling radix sort function
+    radix_sort(arr, n); 
+    
+    //printing the sorted array
+    print(arr, n);
```
Comment (nikochiko) [inline Sorting/radix_sort/RadixSort.cpp.cpp:77]: And please take input from STDIN, 
first line input should be N - number of elements, and second line should be N space-separated numbers

For example - 
```
5   // N
5 9 2 3 10  // N=5 space separated integers
```

This is to make the code standard competition format

**Review by nikochiko — CHANGES_REQUESTED** [review]
Also, avoid plagiarizing code. I understand that it is the same algorithm and there won't be much difference, but at least avoid copying as it is. (Reference: https://www.geeksforgeeks.org/radix-sort/)

**Pull request closed without merging**